src/new_template_sushangagneja.html

```diff
@@ -0,0 +1,60 @@
+
+<body>
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.12/css/all.css" crossorigin="anonymous">
+<link href="https://fonts.googleapis.com/css?family=Cabin|Mansalva|Nunito&display=swap" rel="stylesheet">
+
+<style type="text/css">
+       a.link {margin:0;padding:0;border:none;text-decoration:none;}
+       a {color:#273746;}
+       #sig {font-family: 'Dosis', sans-serif;}
```

Comment:
@sushangagneja This has not been imported.

Reply:
I've removed these. Should I make a new PR?